[2023-02-27 09:25:11,875]-[root]-[ERROR]-[main_server.py:<module>():25 Line]:Traceback (most recent call last):
  File "/home/xu/myworkspace/videoStream/cloud_api_sample/tornado_template/main_server.py", line 15, in <module>
    ws_app = Application()
  File "/home/xu/myworkspace/videoStream/cloud_api_sample/tornado_template/Router/router.py", line 17, in __init__
    (r'/restart', RestartHandler),
NameError: name 'RestartHandler' is not defined

[2023-02-27 09:25:21,301]-[root]-[INFO]-[main_server.py:<module>():22 Line]:AiServer启动成功!
[2023-02-27 09:25:42,537]-[root]-[ERROR]-[main_server.py:<module>():25 Line]:Traceback (most recent call last):
  File "/home/xu/myworkspace/videoStream/cloud_api_sample/tornado_template/main_server.py", line 23, in <module>
    tornado.ioloop.IOLoop.instance().start()
  File "/home/xu/.local/lib/python3.8/site-packages/tornado/platform/asyncio.py", line 199, in start
    self.asyncio_loop.run_forever()
  File "/usr/lib/python3.8/asyncio/base_events.py", line 570, in run_forever
    self._run_once()
  File "/usr/lib/python3.8/asyncio/base_events.py", line 1823, in _run_once
    event_list = self._selector.select(timeout)
  File "/usr/lib/python3.8/selectors.py", line 468, in select
    fd_event_list = self._selector.poll(timeout, max_ev)
KeyboardInterrupt

[2023-02-27 09:25:44,152]-[root]-[INFO]-[main_server.py:<module>():22 Line]:AiServer启动成功!
[2023-02-27 09:25:50,270]-[root]-[ERROR]-[main_server.py:<module>():25 Line]:Traceback (most recent call last):
  File "/home/xu/myworkspace/videoStream/cloud_api_sample/tornado_template/main_server.py", line 23, in <module>
    tornado.ioloop.IOLoop.instance().start()
  File "/home/xu/.local/lib/python3.8/site-packages/tornado/platform/asyncio.py", line 199, in start
    self.asyncio_loop.run_forever()
  File "/usr/lib/python3.8/asyncio/base_events.py", line 570, in run_forever
    self._run_once()
  File "/usr/lib/python3.8/asyncio/base_events.py", line 1823, in _run_once
    event_list = self._selector.select(timeout)
  File "/usr/lib/python3.8/selectors.py", line 468, in select
    fd_event_list = self._selector.poll(timeout, max_ev)
KeyboardInterrupt

[2023-02-27 09:34:35,050]-[root]-[INFO]-[main_server.py:<module>():22 Line]:AiServer启动成功!
[2023-02-27 09:35:29,944]-[root]-[ERROR]-[main_server.py:<module>():25 Line]:Traceback (most recent call last):
  File "/home/xu/myworkspace/videoStream/cloud_api_sample/tornado_template/main_server.py", line 23, in <module>
    tornado.ioloop.IOLoop.instance().start()
  File "/home/xu/.local/lib/python3.8/site-packages/tornado/platform/asyncio.py", line 199, in start
    self.asyncio_loop.run_forever()
  File "/usr/lib/python3.8/asyncio/base_events.py", line 570, in run_forever
    self._run_once()
  File "/usr/lib/python3.8/asyncio/base_events.py", line 1823, in _run_once
    event_list = self._selector.select(timeout)
  File "/usr/lib/python3.8/selectors.py", line 468, in select
    fd_event_list = self._selector.poll(timeout, max_ev)
KeyboardInterrupt

[2023-02-27 09:35:32,730]-[root]-[INFO]-[main_server.py:<module>():22 Line]:AiServer启动成功!
[2023-02-27 09:35:37,591]-[root]-[ERROR]-[main_server.py:<module>():25 Line]:Traceback (most recent call last):
  File "/home/xu/myworkspace/videoStream/cloud_api_sample/tornado_template/main_server.py", line 23, in <module>
    tornado.ioloop.IOLoop.instance().start()
  File "/home/xu/.local/lib/python3.8/site-packages/tornado/platform/asyncio.py", line 199, in start
    self.asyncio_loop.run_forever()
  File "/usr/lib/python3.8/asyncio/base_events.py", line 570, in run_forever
    self._run_once()
  File "/usr/lib/python3.8/asyncio/base_events.py", line 1823, in _run_once
    event_list = self._selector.select(timeout)
  File "/usr/lib/python3.8/selectors.py", line 468, in select
    fd_event_list = self._selector.poll(timeout, max_ev)
KeyboardInterrupt

[2023-02-27 09:36:16,344]-[root]-[INFO]-[main_server.py:<module>():22 Line]:AiServer启动成功!
[2023-02-27 09:36:18,412]-[root]-[ERROR]-[main_server.py:<module>():25 Line]:Traceback (most recent call last):
  File "/home/xu/myworkspace/videoStream/cloud_api_sample/tornado_template/main_server.py", line 23, in <module>
    tornado.ioloop.IOLoop.instance().start()
  File "/home/xu/.local/lib/python3.8/site-packages/tornado/platform/asyncio.py", line 199, in start
    self.asyncio_loop.run_forever()
  File "/usr/lib/python3.8/asyncio/base_events.py", line 570, in run_forever
    self._run_once()
  File "/usr/lib/python3.8/asyncio/base_events.py", line 1823, in _run_once
    event_list = self._selector.select(timeout)
  File "/usr/lib/python3.8/selectors.py", line 468, in select
    fd_event_list = self._selector.poll(timeout, max_ev)
KeyboardInterrupt

[2023-02-27 09:36:26,044]-[root]-[INFO]-[main_server.py:<module>():22 Line]:AiServer启动成功!
[2023-02-27 09:38:02,396]-[root]-[ERROR]-[main_server.py:<module>():25 Line]:Traceback (most recent call last):
  File "/home/xu/myworkspace/videoStream/cloud_api_sample/tornado_template/main_server.py", line 23, in <module>
    tornado.ioloop.IOLoop.instance().start()
  File "/home/xu/.local/lib/python3.8/site-packages/tornado/platform/asyncio.py", line 199, in start
    self.asyncio_loop.run_forever()
  File "/usr/lib/python3.8/asyncio/base_events.py", line 570, in run_forever
    self._run_once()
  File "/usr/lib/python3.8/asyncio/base_events.py", line 1823, in _run_once
    event_list = self._selector.select(timeout)
  File "/usr/lib/python3.8/selectors.py", line 468, in select
    fd_event_list = self._selector.poll(timeout, max_ev)
KeyboardInterrupt

[2023-02-27 09:38:04,231]-[root]-[INFO]-[main_server.py:<module>():22 Line]:AiServer启动成功!
[2023-02-27 09:39:12,775]-[root]-[ERROR]-[main_server.py:<module>():25 Line]:Traceback (most recent call last):
  File "/home/xu/myworkspace/videoStream/cloud_api_sample/tornado_template/main_server.py", line 23, in <module>
    tornado.ioloop.IOLoop.instance().start()
  File "/home/xu/.local/lib/python3.8/site-packages/tornado/platform/asyncio.py", line 199, in start
    self.asyncio_loop.run_forever()
  File "/usr/lib/python3.8/asyncio/base_events.py", line 570, in run_forever
    self._run_once()
  File "/usr/lib/python3.8/asyncio/base_events.py", line 1823, in _run_once
    event_list = self._selector.select(timeout)
  File "/usr/lib/python3.8/selectors.py", line 468, in select
    fd_event_list = self._selector.poll(timeout, max_ev)
KeyboardInterrupt

